packages/collabswarm-subtlecrypto/package.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
@lifekaizen Is this package/module still used?

Reply:
Will remove!